src/Main.ts

```diff
@@ -23,7 +24,7 @@ async function start() {
     // Glob string for loading classes
     classes: [`./commands/*.{js,ts}`],
     // If you only want to use guild commands, uncomment this line
-    // botGuilds: [(client) => client.guilds.cache.map((guild) => guild.id)],
+    botGuilds: process.env.NODE_ENV ? ["930471643594240023"] : undefined,
```

Comment:
What GuildID is this?

Reply:
oops, overlooked...
it's the GuildId of the funni bot server, where I tested the bot

Reply:
since it's array we could probably also add in our server guild ID
that way we can test it on both servers, it's a winwin

Reply:
+ we kinda need this botGuilds attribute to make the bot automatically load in all slash commands once it runs

Reply:
Aight, I'll add that in instead...
just noticed an error in the README file aswell, so I'll fix that

Reply:
I did some testing, with this option enabled like this, it automatically adds the commands to the servers it's connected to...
The command is basically doing the same thing as we are doing when hardcoding... so it's your call...

Reply:
ok then let's leave it the way it was

Reply:
👍👍👍